src/components/shared/forms/styles/formControl.scss

```diff
@@ -52,25 +52,25 @@ $textareaHeight: 60px;
         background: rgba(themed('colorSystem'), .99);
       }
 
-      &::-webkit-input-placeholder { color: themed('colorLightestSecond'); }
-          &:-ms-input-placeholder { color: themed('colorLightestSecond'); }
-              &::-moz-placeholder { color: themed('colorLightestSecond'); }
-                &:-moz-placeholder { color: themed('colorLightestSecond'); }
+      &::-webkit-input-placeholder { color: themed('colorInputPlaceholderText'); }
+          &:-ms-input-placeholder { color: themed('colorInputPlaceholderText'); }
+              &::-moz-placeholder { color: themed('colorInputPlaceholderText'); }
+                &:-moz-placeholder { color: themed('colorInputPlaceholderText'); }
```

Comment:
nit: The tabs don't seem consistent is that expected?

Reply:
nope, this was left over from simulation, fixed